mount/driver/build.rs

```diff
@@ -9,12 +9,12 @@ fn main() {
         Ok(val) => match val.as_str() {
             "MOUNT_PDR_8kHz" => "sys_pdr_8k-linear",
             "MOUNT_FDR_1kHz" | "MOUNT_FDR_1kHz-az17Hz"=> "sys_fdr_1k",
-            //"MOUNT_FDR_8kHz" => "sys_fdr_8k",
+            "MOUNT_FDR_8kHz" => "sys_fdr_8k",
             value => panic!("expected one of the following `MOUNT_MODEL`: `MOUNT_PDR_8kHz`, `MOUNT_FDR_1kHz` or `MOUNT_FDR_1kHz-az17Hz`, found {value}"),
```

Comment:
can you add MOUNT_FDR_8kHz to the list of MOUNT_MODELs

Reply:
Commit [7d79522](https://github.com/rconan/gmt-controllers/pull/8/commits/7d79522357d274d6423cf55e1a1e4862b066a13a) addresses  that.